src/components/Utilities/TypographyUsage.css

```diff
@@ -44,6 +44,10 @@
   @mixin eds-typography-preset-009-bold;
 }
 
+.u-theme-typography-body-text-xl {
```

Comment:
Add CSS Comment

Reply:
Not sure how I would comment on typography usage classes, as there are no comments on the other classes as well?